svf/include/MSSA/SVFGBuilder.h

```diff
@@ -49,7 +49,10 @@ class SVFGBuilder
     typedef SVFG::SVFGEdgeSetTy SVFGEdgeSet;
 
     /// Constructor
-    explicit SVFGBuilder(bool _SVFGWithIndCall = false): svfg(nullptr), SVFGWithIndCall(_SVFGWithIndCall) {}
+    explicit SVFGBuilder(bool _SVFGWithIndCall = false, bool _OptimiseSVFG = false) :
```

Comment:
Could we change the second argument to

bool _OptimiseSVFG = Options::OPTSVFG()

Also, please use a different name instead of `OptimiseSVFG` to avoid a readability conflict with `void optimiseSVFG()`.